src/nuget/Microsoft.DotNet.BuildTools.Run.nuspec

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.Run</id>
+    <version>1.0.0-prerelease</version>
+    <title>Microsoft DotNet Run Tool</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=518631</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <summary>.NET Core Framework Run Tool</summary>
+    <description>This package provides build-time support for the .NET Core Framework libraries.  You should not need to reference this package from your project.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags></tags>
+    <dependencies>
+    <!--
+      Do not add any dependencies to this package as we want to install it versionless
+      (i.e. -ExcludeVersion) and we don't want the dependencies to be installed that way
+    -->
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="run\run.exe" target="tools\netcoreapp1.0" />
```

Comment:
OK. I'd be interested in having a look at the bootstrapping code you use will use to start consuming the run tool. 

Another challenge we have been having is agreeing upon the versions of the CLI, because the bootstrapping code has to agree with the version in the runtimeconfig.json file in order to run against a shared framework (or we have to do something like buy-pass the muxer and go directly into the shared framework ourselves, in which case we don't need a runtimeconflig.json file).